cl/opinion_page/templates/includes/authorities_section.html

```diff
@@ -0,0 +1,33 @@
+{% load humanize %}
+{% load extras %}
+{% load humanize %}
+{% load static %}
+{% load text_filters %}
+
+    {# Show cases this case cites #}
+        {% if authorities_context.total_authorities_count > 0  %}
```

Comment:
Nit: I'd outdent these lines all the way to the left (and subsequent lines accordingly):

```suggestion
{# Show cases this case cites #}
{% if authorities_context.total_authorities_count > 0  %}
```

And I guess while we're at it, we should do two spaces, not four, as per the `.editorconfig` file.